examples/tutorials/advanced/draping_on_3d_surface.py

```diff
@@ -0,0 +1,181 @@
+
+# Download an PNG image of the flag of the EU using rasterio and load it into a
+# xarray.DataArray
+url_to_image = "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b7/Flag_of_Europe.svg/1000px-Flag_of_Europe.svg.png"
+with rasterio.open(url_to_image) as dataset:
+    data = dataset.read()
+    drapegrid = xr.DataArray(data, dims=("band", "y", "x"))
+
+# %%
+# Again we create a 3-D plot with :meth:`pygmt.Figure.grdview` and passe an Earth relief
+# grid to the ``grid`` parameter to create the topographic surface. But now we pass the
+# PNG image which was loaded into an :class:`xarray.DataArray` to the ``drapgrid``
+# parameter.
+
+fig = pygmt.Figure()
+
+# Set up a colormap with two colors for the EU flag: blue (0/51/153) for the background
+# (value 0 in the nedCDF file -> lower half of 0-255 range) and yellow (255/204/0) for
+# the stars (value 255 -> upper half)
+pygmt.makecpt(cmap="0/51/153,255/204/0", series=[0, 256, 128])
+
+fig.grdview(
+    projection="M12c",  # Mercator projection with a width of 12 centimeters
+    region=region_3d,
+    grid=grd_relief,  # Use elevation grid for z values
+    drapegrid=drapegrid,  # Drap image grid for the EU flag on top
+    cmap=True,  # Use colormap defined for the EU flag
+    surftype="i",  # Create an image plot
+    # Use an illumination from the azimuthal directions 0° (north) and 270° (west) with
+    # a normalization via a cumulative Laplace distribution for the shading
+    shading="+a0/270+ne0.6",
+    perspective=[157.5, 30],  # Define azimuth, elevation for the 3-D plot
+    zsize="1c",
+    plane="+gdarkgray",
+    frame=True,
+)
+
+fig.show()
+
+# %%
+# Additionally we can plot some features like coastlines, symbols, and text on top of
+# the map. Setting ``perspective=True`` leads to the same azimuth and elevation values
+# as we passed to the ``perspective`` parameter of :meth:`pygmt.Figure.grdview`.
+
+# Plot water masses, political broders, and shorelines
+fig.coast(
+    water="white@50",
+    borders="1/1p,lightgray",
+    shorelines="1/0.5p,gray30",
+    perspective=True,
+)
+
+# Set up a pandas.DataFrame with coordinates and names of three cities
+cities = pd.DataFrame(
+    {
+        "longitude": [7.10, 4.35, 5.69],  # degrees East
+        "latitude": [50.73, 50.85, 50.85],  # degress North
+        "elevation": [60, 13, 49],  # meters
+        "name": ["Bonn", "Bruxelles", "Maastricht"],
+    }
+)
+# Plot markers
+fig.plot3d(
+    x=cities.longitude,
+    y=cities.latitude,
+    z=cities.elevation,
+    style="s0.3c",  # Use squares with a size of 0.3 centimeters
+    pen="1.5p,white",
+    fill="black",
+    perspective=True,
+)
+# Add labels
+fig.text(
+    x=cities.longitude,
+    y=cities.latitude,
+    text=cities.name,
+    justify="TL",  # Use Top Left corner as anchor point
+    offset="0.3c/-0.3c",  # x / y directions, in centimeters
+    font="12p",
+    fill="white@30",  # Fill box in white with a transparency of 30 %
+    perspective=True,
+)
```

Comment:
Maybe we should skip this part and only focus on draping?

Reply:
I apologize for the delay with this tutorial 🙁!

I also thought about this. Maybe we can remove this part for now, to get this tutorial into the release of v0.13.0. 

However, I am not 100 % happy with this, as I feel that users actually like to plot more details on top of the 3-D surface. Maybe we can add this later or create a separate tutorial on how to do this.

Reply:
> as I feel that users actually like to plot more details on top of the 3-D surface. Maybe we can add this later or create a separate tutorial on how to do this.

Sounds good to me.